Quiz/src/main/kotlin/furhatos/app/quiz/questionlist.kt

```diff
@@ -0,0 +1,89 @@
+package furhatos.app.quiz
+
+/**
+ * The questions are structured like
+ *  -The question
+ *  -The correct answer, followed by alternative pronounciations
+ *  -A list of other answers followed by their alternatives
+ */
```

Comment:
Should we used named parameters here to make this easier to read?

Reply:
Named!